front/src/modules/timetableItem/hooks/useSelectedTimetableItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import type { TimetableItem } from 'reducers/osrdconf/types';
+import { getSelectedTrainId } from 'reducers/simulationResults/selectors';
+import { extractPacedTrainIdFromOccurrenceId, isOccurrenceId } from 'utils/trainId';
+
+const useSelectedTimetableItem = (): TimetableItem | undefined => {
+  const trainId = useSelector(getSelectedTrainId);
+
+  const timetableItemId = useMemo(() => {
+    if (!trainId) return undefined;
+    return isOccurrenceId(trainId) ? extractPacedTrainIdFromOccurrenceId(trainId) : trainId;
+  }, [trainId]);
+
+  const { currentData: timetableItem } = osrdEditoastApi.endpoints.getTimetableItemById.useQuery(
+    {
+      id: timetableItemId!,
+    },
+    { skip: !timetableItemId }
```

Comment:
i was wondring because i found it recently, why not using the built-in skipToekn from rtk query ?
https://redux-toolkit.js.org/rtk-query/usage-with-typescript#skipping-queries-with-typescript-using-skiptoken